Flatr-App-Core.cabal

```diff
@@ -36,7 +36,8 @@ executable Flatr-App-Core-exe
                      , word8
                      , bytestring
                      , base16-bytestring >=0.1
-                     , cryptohash >=0.11
+                     , cryptonite >=0.24
```

Comment:
Please add this dependency to the nix file.

Reply:
Fixed in Commit e4efd16.